include/makeshift/detail/iomanip.hpp

```diff
@@ -30,22 +30,24 @@ struct manipulator : ToStreamFuncT, FromStreamFuncT
         friend std::ostream&
         operator <<(std::ostream& stream, bound_manipulator&& m)
         {
+            using T = ToStreamFuncT;
```

Comment:
Could you please add a comment stating the reason for introducing a type alias so it doesn't get lost in a refactoring?

Reply:
Added a comment, is it sufficient?

Reply:
Yes, thanks.